frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts

```diff
@@ -18,7 +18,6 @@ import {
 import {createLocalInjector} from "core-app/modules/fields/edit/editing-portal/edit-form-portal.injector";
 import {IFieldSchema} from "core-app/modules/fields/field.base";
 import {EditFieldService, IEditFieldType} from "core-app/modules/fields/edit/edit-field.service";
-import {WorkPackageChangeset} from "core-components/wp-edit/work-package-changeset";
 import {ResourceChangeset} from "core-app/modules/fields/changeset/resource-changeset";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e1c1b00c5a6c973508c7538a865d197276aa638c/frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts#L23"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>